stream_alert_cli/helpers.py

```diff
@@ -154,6 +152,107 @@ def tf_runner(**kwargs):
+            },
+            's3': {
+                'configurationId': 'testConfigRule',
+                'object': {
+                    'eTag': '0123456789abcdef0123456789abcdef',
+                    'sequencer': '0A1B2C3D4E5F678901',
+                    'key': '<TO_BE_REPLACED>',
+                    'size': '<TO_BE_REPLACED>'
+                },
+                'bucket': {
+                    'arn': '<TO_BE_REPLACED>',
+                    'name': '<TO_BE_REPLACED>',
+                    'ownerIdentity': {
+                        'principalId': 'EXAMPLE'
+                    }
+                },
+                's3SchemaVersion': '1.0'
+            },
+            'responseElements': {
+                'x-amz-id-2': 'EXAMPLE123/5678abcdefghijklambdaisawesome/mnopqrstuvwxyzABCDEFGH',
+                'x-amz-request-id': 'EXAMPLE123456789'
+            },
+            'awsRegion': 'us-east-1',
+            'eventName': 'ObjectCreated:Put',
+            'userIdentity': {
+                'principalId': 'EXAMPLE'
+            },
+            'eventSource': 'aws:s3'
+        }
+
+    elif service == 'kinesis':
+        return {
+            'eventID': 'shardId-000000000000:49545115243490985018280067714973144180062593244200961',
+            'eventVersion': '1.0',
+            'kinesis': {
+                'approximateArrivalTimestamp': 1428537600,
+                'partitionKey': 'partitionKey-3',
+                'data': '<TO_BE_REPLACED>',
+                'kinesisSchemaVersion': '1.0',
+                'sequenceNumber': '49545115243490985018280067714973144582180062593244200961'
+            },
+            'invokeIdentityArn': 'arn:aws:iam::EXAMPLE',
+            'eventName': 'aws:kinesis:record',
+            'eventSourceARN': '<TO_BE_REPLACED>',
+            'eventSource': 'aws:kinesis',
+            'awsRegion': 'us-east-1'
+        }
+
+    elif service == 'sns':
+        return {
+            'EventVersion': '1.0',
+            'EventSubscriptionArn': '<TO_BE_REPLACED>',
+            'EventSource': 'aws:sns',
+            'Sns': {
+                'SignatureVersion': '1',
+                'Timestamp': '1970-01-01T00:00:00.000Z',
+                'Signature': 'EXAMPLE',
+                'SigningCertUrl': 'EXAMPLE',
+                'MessageId': '95df01b4-ee98-5cb9-9903-4c221d41eb5e',
+                'Message': '<TO_BE_REPLACED>',
+                'MessageAttributes': {
+                    'Test': {
+                        'Type': 'String',
+                        'Value': 'TestString'
+                    },
+                    'TestBinary': {
+                        'Type': 'Binary',
+                        'Value': 'TestBinary'
+                    }
+                },
+                'Type': 'Notification',
+                'UnsubscribeUrl': 'EXAMPLE',
+                'TopicArn': 'arn:aws:sns:EXAMPLE',
+                'Subject': 'TestInvoke'
+            }
+        }
+
+    elif service == 'stream_alert_app':
```

Comment:
I kind of wish this was just called `lambda`, any reason why it isn't?

Reply:
The only reason I can give is that we control what the incoming payload looks like (where as with sns/s3/kinesis/etc it is outside of our control). I'm not overly opposed to saying lambda, but it seem more clear to use this instead of calling it lambda one place and `stream_alert_app` another place

Reply:
oh right, that's why.